pages/merchant/Merchant.qml

```diff
@@ -265,7 +265,7 @@ Item {
                     font.pixelSize: 12
                     font.bold: false
                     color: "white"
-                    text: "<style type='text/css'>a {text-decoration: none; color: #FF6C3C; font-size: 12px;}</style>Currently selected address: " + addressLabel + " <a href='#'>(Change)</a>"
+                    text: qsTr("<style type='text/css'>a {text-decoration: none; color: #FF6C3C; font-size: 12px;}</style>Currently selected address: ") + addressLabel + qsTr(" <a href='#'>(Change)</a>") + translationManager.emptyString
```

Comment:
In general, we should avoid including code (HTML, css, JavaScript) in translated strings. It makes translations messy & code changes more difficult / error-prone.

We can fix later to avoid breaking existing translations.

Reply:
Already pointed to this issue long time ago. Though the decision was "let's keep it that way for now".

Yes, i agree and i'm sure we should strip all the **garbage** from translated strings.

Reply:
Yeah, I've done a lot of cleanup like this in the past... One can massage the translations and qml such that existing translated strings still match up.